recipes/cpp-subprocess/all/conanfile.py

```diff
@@ -0,0 +1,26 @@
+from conans import ConanFile, tools
+
+
+class CppSubprocess(ConanFile):
+    name = "cpp-subprocess"
```

Comment:
Since this is a fork I think it's helpful to include the author/owner name for people searching

```suggestion
    name = "nextsilicon-cpp-subprocess"
```